.env

```diff
@@ -0,0 +1,4 @@
+# Developer level configuration settings
+
+# Set to "production" for a production server and enviornment, or "development" for local server
```

Comment:
```suggestion
# Set to "production" for a production server and environment, or "development" for local server
```
fix typo

Reply:
Spell check wasn't enabled for .env, fixed now